src/utils/version.ts

```diff
@@ -31,3 +32,12 @@ export async function logOutdatedVersion() {
         );
     }
 }
+
+export function checkNodeMinimumVersion() {
+    if (compare(process.version, NODE_MINIMUM_VERSION, "<")) {
```

Comment:
I've noticed that we use two libraries that kind of do the same thing: https://www.npmjs.com/package/semver and https://www.npmjs.com/package/compare-versions.

Not sure if we should do something now about this.